public/index.html

```diff
@@ -10,6 +10,8 @@
       homescreen on Android. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js?config=TeX-MML-AM_HTMLorMML-full&amp;delayStartupUntil=configured" async></script>
```

Comment:
Oh nice, there is some page jitter but maybe MathJax corrects itself after rendering? For example, http://localhost:3000/books/Ax2o07Ul/pages/M_qlK4M9#63848_copy_1 points to "10.4 Rotational Kinetic Energy: Work and Energy Revisited" and after some jitter, continues to point to the same place.

Hmm, what happens if we have a floating (absolute positioned) bar at the top? Will MathJax's `scrollTo` cause content to no longer be visible?

Also, any thoughts on reducing the jitter in the future? Some options: 

- wrap each `<math>` with a `<span/div style="display: block; width: ###; height: ###">`
- prerender using MathJax and then re-render in the browser (would need to squirrel away the MathML)

Reply:
yea we have openstax/unified-issues#53 to try out pre-rendering math, i'm not sure how its going to work out but i think that would be ideal